lib/ffaker/date.rb

```diff
@@ -27,5 +27,14 @@ def forward(days = 365)
 
       between(from, to)
     end
+
+    # Random birthday date (maximum age between 18 and 65)
+    # Keyword arguments: min_age, max_age
+    def birthday(min_age: 18, max_age: 65)
+      from = ::Date.today - 365 * max_age
+      to   = ::Date.today - 365 * min_age
```

Comment:
- In this implementation, birthday(min_age: 42, max_age:42) is expected to be the same range of dates.
- This implementation ignores leap years.
```suggestion
      from = ::Date.today.prev_year(max_age)
      to = ::Date.today.prev_year(min_age - 1). prev_day
```